driver/node.go

```diff
@@ -182,7 +182,7 @@ func (s *nodeService) nodePublishFilesystemVolume(req *csi.NodePublishVolumeRequ
 		}
 	}
 
-	err = os.MkdirAll(req.GetTargetPath(), 0755)
+	err = os.MkdirAll(req.GetTargetPath(), 2777)
```

Comment:
What happens if the target directory already exists?
Shouldn't we use `os.Chmod` for that case?

Reply:
This is changing the mode of directory _before_ mount.
What we need to do is to change the directory mode _after_ mount.